ios/PiwikProSdk.m

```diff
@@ -92,4 +98,13 @@ @implementation PiwikProSdk
   }
 }
 
+- (void)setCustomDimensions:(NSArray *)customDimensions {
+    for (NSDictionary* customDimension in customDimensions) {
```

Comment:
`customDimensions` can be `null` here, but apparently the code works without explicit checks. Wonder if it would be good to add `nullable` annotation to make it clearer.